app/src/testDebug/java/io/reark/rxgithubapp/utils/StethoInstrumentationTest.java

```diff
@@ -76,6 +76,5 @@ public void testDecorateNetwork() {
 
         instrumentation.decorateNetwork(okHttpClient);
 
-        verify(instrumentation).addInterceptor(eq(okHttpClient), eq(interceptor));
```

Comment:
This was the verify part of the test function, this won't now do anything without it